docs/api/modules/App-Configuration.md

```diff
@@ -176,13 +176,17 @@ In practice, this allows POST requests from given origins to return partially re
 * ✅ Root Module
 * 🚫 Child Module
 
+> The PWA feature is behind a feature flag `ONE_SERVICE_WORKER` that needs to be
```

Comment:
#style-nitpick Emoji prefix for the callout? Helps draw the eye.

Reply:
Good call, updated 4492fbc